docker-compose.yml

```diff
@@ -36,20 +38,22 @@ services:
     depends_on:
       - mongodb
       - redis
+    container_name: ws
     build:
       context: .
       dockerfile: Dockerfile.ws
     environment:
       NODE_ENV: ${NODE_ENV}
       MONGO_URL: ${MONGO_URL}
-      REDIS_HOST: redis
+      REDIS_HOST: ${REDIS_HOST}
```

Comment:
@djabarovgeorge do we need to add all the variables mapping from .example.env to here or they just work like this? I think we explicitly need to add things like `S3_LOCAL_STACK` here for it to work.

Reply:
Yes, we need to add the mapping, I will add more on the next commit. 🙂